docs/modules/advanced_speed_servo_info.rst

```diff
@@ -37,6 +38,7 @@ Supported IQUART Clients
         * :ref:`arming_handler`
         * :ref:`stopping_handler`
         * :ref:`iquart_flight_controller_interface`
+        * :ref:`throttle_source_manager`
```

Comment:
Maybe worth saying that its v0.1.0 and above that supports it to be clear here? Now that they are nicely synced up on their versions